### PR TITLE
Add FAQ content to Kanso (New UI) doc

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Add FAQ content to New UI doc

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Review by agaur — APPROVED** [review]

File: docs/get-started/sumo-logic-ui-new.md [hunk 1]
```diff
@@ -211,6 +211,85 @@ You don't have to stop there either. You can take the next step and become Sumo
...
+
+You have the flexibility to opt out whenever you choose. Just click the **Return to Classic UI** option on your left navigation menu of the New UI. Once you've switched back to the Classic UI, all subsequent sessions will default to the Classic UI experience.
+
+If you enter the New UI URL in your browser, it will still direct you to the New UI. We have implemented a translation from the old (Classic) UI to the New UI, but not the other way around.
+
+### When is Sumo Logic planning to remove access to the Classic UI?
+
+Our goal is to turn off access to the old (Classic) UI at the end of 2024. However, this timeline is subject to our successful Beta testing.
+
+### Are there certain capabilities that are not going to be available during Beta?
```
Comment (agaur) [inline docs/get-started/sumo-logic-ui-new.md:280]: @kimsauce we might have to make some additions to this section. But it's good for first round.

Comment (kimsauce) [reply docs/get-started/sumo-logic-ui-new.md:280]: Sounds good!

File: docs/get-started/sumo-logic-ui-new.md [hunk 2]
```diff
@@ -211,6 +211,86 @@ You don't have to stop there either. You can take the next step and become Sumo
...
 
+## FAQ
+
+This guide offers responses to frequently asked questions regarding the Sumo Logic UI redesign project, which involves transitioning from the current Classic UI to the upcoming New UI.
+
+### What's being launched?
+
+We are excited to introduce the Sumo Logic Unified Experience, also known as Project Kanso, inspired by the Japanese principle of simplicity and clutter elimination. This initiative integrates the capabilities of our Log Analytics, Cloud SIEM, and Cloud SOAR into a unified navigation system. Alongside this integration, we have implemented several user interface enhancements to make all Sumo Logic features more accessible and user-friendly.
+
+### What issues does the New UI resolve?
```
[the remark this answers is not in the document]
Comment (jpipkin1) [reply docs/get-started/sumo-logic-ui-new.md:222]: Did you mean to present this as a proper noun? If so, leave as-is.

File: docs/get-started/sumo-logic-ui-new.md [hunk 3]
```diff
@@ -211,6 +211,86 @@ You don't have to stop there either. You can take the next step and become Sumo
...
+
+If you prefer having all Sumo Logic tabs in one place - without getting mixed with other browser tabs - we suggest utilizing the [tab grouping functionality](#customize-your-environment-withtabs) offered by most browsers. By adding your Sumo Logic tab to a tab group, any new tabs opened within the Sumo Logic platform will automatically open in the same tab group. You can also collapse the tab group to reclaim valuable real estate in the browser's tab bar.
+
+### Will the New UI continue the functionality of remembering the tabs I had opened in my previous session, similar to the Classic UI?
+
+After analyzing tab usage data, we've found that a minimal portion of previously opened tabs are actively utilized by our users. Consequently, the Sumo Logic UI often remains cluttered with multiple unused tabs. With the introduction of the New UI experience, if you fail to close browser tabs from previous sessions, they will automatically reload upon login.
+
+<img src={useBaseUrl('img/get-started/tab-reload.gif')} alt="tab-reload.gif" />
```
Comment (jpipkin1) [inline docs/get-started/sumo-logic-ui-new.md:261]: This gif needs to move up into the previous section since it shows tab grouping.

**Review by jpipkin1 — APPROVED** [review]
Perfect, @kimsauce.

**Comment by kimsauce** [comment]
> Perfect, @kimsauce.

thank you for the quick review!

**Pull request merged**